shared/helpers.js

```diff
@@ -70,3 +71,15 @@ export function translate(format, bindings = {}) {
 
     return ret;
 }
+
+export function currencyExists(currency) {
+    return typeof currencies.find(curr => curr.symbol === currency) !== 'undefined';
+}
+
+export function formatCurrency(currency) {
+    return amount => currencyFormatter(amount, { code: currency });
+}
+
+export function getCurrencySymbol(currency) {
+    return findCurrency(currency).symbol;
```

Comment:
Is there a chance `findCurrency` can be used in `isKnownCurrency` too? Also, can you rename it `currencySymbolFor`?

Or even better:
- put these three functions in an object called `currency` and that gets exported.
- rename these three functions as `isKnown`, `makeFormat` and `symbolFor`

What do you think? Call sites will read `currency.isKnown(currency);` and such, which is cute.